app/viewmodels/journeys/taskList/ExpensesTasklist.scala

```diff
@@ -361,18 +286,22 @@ object TradeJourneyStatusesViewModel {
       .onPageLoad(taxYear, businessId, NormalMode)
       .url
     journeyStatus match {
-      case Completed | InProgress                                         => cyaUrl
-      case NotStarted | CheckOurRecords if wfhTailoring contains Yes      => wfhUrl
-      case NotStarted | CheckOurRecords if wfbpTailoring exists (_ != No) => wfbpUrl
-      case _                                                              => "#"
+      case Completed | InProgress                                                                  => cyaUrl
+      case NotStarted | CheckOurRecords if wfhTailoring contains WorkFromHome.Yes                  => wfhUrl
+      case NotStarted | CheckOurRecords if wfbpTailoring exists (_ != WorkFromBusinessPremises.No) => wfbpUrl
+      case _                                                                                       => "#"
     }
   }
 
-  private def determineUrl(startUrl: String, cyaUrl: String)(implicit status: JourneyStatus): String =
-    status match {
-      case CannotStartYet               => "#"
-      case Completed | InProgress       => cyaUrl
-      case NotStarted | CheckOurRecords => startUrl
-    }
+  private def conditionPassedForViewableLink[A](page: OneQuestionPage[A], acceptableAnswers: Seq[A])(implicit
```

Comment:
Some of these may be moved to the package object if needed for Capital Allowance as well